spec.html

```diff
@@ -19023,12 +19067,15 @@ <h2>Syntax</h2>
     </emu-grammar>
 
     <emu-clause id="sec-directive-prologues-and-the-use-strict-directive">
-      <h1>Directive Prologues and the Use Strict Directive</h1>
+      <h1>Directive Prologues</h1>
       <p>A <dfn id="directive-prologue">Directive Prologue</dfn> is the longest sequence of |ExpressionStatement|s occurring as the initial |StatementListItem|s or |ModuleItem|s of a |FunctionBody|, a |ScriptBody|, or a |ModuleBody| and where each |ExpressionStatement| in the sequence consists entirely of a |StringLiteral| token followed by a semicolon. The semicolon may appear explicitly or may be inserted by automatic semicolon insertion. A Directive Prologue may be an empty sequence.</p>
       <p>A <dfn id="use-strict-directive">Use Strict Directive</dfn> is an |ExpressionStatement| in a Directive Prologue whose |StringLiteral| is either of the exact code point sequences `"use strict"` or `'use strict'`. A Use Strict Directive may not contain an |EscapeSequence| or |LineContinuation|.</p>
-      <p>A Directive Prologue may contain more than one Use Strict Directive. However, an implementation may issue a warning if this occurs.</p>
+      <p>A <dfn id="sensitive-directive">Sensitive Directive</dfn> is an |ExpressionStatement| in a Directive Prologue whose |StringLiteral| is either of the exact code point sequences `"sensitive"` or `'sensitive'`. A Sensitive Directive may not contain an |EscapeSequence| or |LineContinuation|.</p>
```

Comment:
why are any of these dfn ids necessary?

Reply:
When ecmarkup auto-links occurrences of defined terms:
- For a `<dfn>` with an `id`, the link will go to the `<dfn>`.
- For a `<dfn>` without an `id`, the link will go to the `<dfn>`'s containing clause. (I.e., ecmarkup doesn't fabricate an id.)

So whether the ids are necessary depends on how precise the links need to be. (Personally, I'm okay with the latter if the `<dfn>` is close to the start of the clause.)